pkgs/applications/misc/bukut/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, python3, fetchFromGitHub }:
+
+with python3.pkgs; buildPythonApplication rec {
+  version = "0.11";
+  pname = "bukut";
+
+  src = fetchFromGitHub {
+    owner = "peterjschroeder";
+    repo = "bukut";
+    rev = "9d735490dae9f657a74f33f2eb68656b31a5c5d8";
+    sha256 = "sha256-MCAyMvzXnC7QSpWeG7qorK4JmRBMzG3eGsp5YpKmVjg=";
+  };
+
+  propagatedBuildInputs = [
+    asciimatics
+    beautifulsoup4
+    natsort
+    pyperclip
+    pyxdg
+  ];
+
+  meta = with lib; {
+    description = "Text user interface for buku bookmark manager";
+    homepage = "https://github.com/peterjschroeder/bukut";
+    platforms = platforms.unix;
```

Comment:
Set by buildPythonApplication

Reply:
What is set by buildPythonApplication? The platforms? I can remove that line if so.

Reply:
Fixed